pipeline_dp/dp_engine.py

```diff
@@ -153,7 +153,7 @@ def _aggregate(self, col, params: pipeline_dp.AggregateParams,
                 # This regime assumes the input data doesn't have privacy IDs,
                 # and therefore we didn't group by them and cannot guarantee one
                 # row corresponds to exactly one privacy ID.
-                max_rows_per_privacy_id = params.max_contributions_per_partition
+                max_rows_per_privacy_id = params.max_contributions or params.max_contributions_per_partition
```

Comment:
Why does max_contributions takes precedence over max_contributions_per_partition? Given this is used for partition selection, don't we always want to use max_contributions_per_partition, regardless of how many total contributions there are?